javascript/javascript3/week3/classWork/inde.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>javascript-3 week-3 class work</title>
+    <title>javascript-3 week-3 class work </title>
 </head>
```

Comment:
I think I already did a review for it, but it is another pull request

Reply:
True you already reviewed it But I have merged before the mentor 's review so I have created again